index.html

```diff
@@ -29,6 +36,7 @@
   <script src="../storage.js"></script>
   <link rel="stylesheet" href="src/factory.css">
   <script src="https://cdn.rawgit.com/google/code-prettify/master/loader/run_prettify.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
Can this be pulled in via package.json, so the app will work without network access?